test/integration/test-fixtures/image-owning-package/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM ubuntu:20.04
+# this covers rpm-python
+RUN apt update && apt install -y python-pil
```

Comment:
It could be good to pin installed dependencies. We've run into a few issues where lack of pinning meant our test outcomes changed unexpectedly down the road after a completely unrelated change